handlers/templates/pages/guest-link-index.html

```diff
@@ -82,6 +86,60 @@
           });
       });
     });
+
+    const guestLinkButtonStates = {
+      Enable: {
+        apiCall: guestLinkEnable,
+        message: "Guest link enabled successfully.",
+        buttonClass: "is-sucess",
+        iconClass: "fa-check-square",
+        nextState: "Disable",
+      },
+      Disable: {
+        apiCall: guestLinkDisable,
+        message: "Successfully disabled the guest link.",
```

Comment:
Can we make this "Guest link disabled" to match the structure of the "enable" message?